themes/survey/vanilla/views/subviews/header/google_analytics.twig

```diff
@@ -15,26 +15,46 @@
     See: survey settings
 #}
 {% if aSurveyInfo.googleanalyticsapikey  is defined and aSurveyInfo.googleanalyticsapikey and aSurveyInfo.googleanalyticsstyle %}
-    {% if aSurveyInfo.googleanalyticsstyle  is defined and aSurveyInfo.googleanalyticsstyle == 1 %}
-        <script>
-        (function(i,s,o,g,r,a,m){ i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments) },i[r].l=1*new Date();a=s.createElement(o),
-            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    {# Check if the tracking ID is a Universal Analytics (legacy) or a Google Analytics 4 ID #}
+    {% if aSurveyInfo.googleanalyticsapikey starts with 'UA-' %}
```

Comment:
:+1:

Reply:
@Shnoulle @gabrieljenik can we please merge this into the next 3.x release? We would love to use this for a new project starting next week.